src/es/components/molecules/tooltip/Tooltip.js

```diff
@@ -0,0 +1,190 @@
+// @ts-check
+import { Shadow } from '../../web-components-toolbox/src/es/components/prototypes/Shadow.js'
+
+/**
+* @export
+* @class Tooltip
+* @type {CustomElementConstructor}
+*/
+export default class Tooltip extends Shadow() {
+  constructor (options = {}, ...args) {
+    super({ importMetaUrl: import.meta.url, ...options }, ...args)
+  }
+
+  connectedCallback () {
+    if (this.shouldRenderCSS()) this.renderCSS()
+
+    this.text = this.getAttribute('text') || 'This is a fallback tooltip text!';
+
+    if (this.shouldRenderHTML()) this.renderHTML()
+
+    this.toggleTooltip();
+  }
+
+  disconnectedCallback () {
+    this.root.removeEventListener('click');
+  }
+
+    /**
+   * Toggle tooltip
+   */
+  toggleTooltip() {
+    const tooltip = this.root.querySelector('.tooltip');
+
+    this.root.addEventListener('click', () => {
```

Comment:
ich würde den addEventListener in den "connectedCallback" nehmen und den Event mit removeEventListener im "disconnectedCallback" removen. Somit hast du dann nicht die EventListener doppelt, falls mal die Komponente disconnected und connected. Was dann zu doppel toggle führen würde.